src/main/java/org/aksw/gerbil/dataset/impl/generic/SpanComparator.java

```diff
@@ -0,0 +1,20 @@
+package org.aksw.gerbil.dataset.impl.generic;
+
+import java.util.Comparator;
+
+import org.aksw.gerbil.transfer.nif.Span;
+
+public class SpanComparator implements Comparator<Span>{

```

Comment:
Please rename it to something like `LengthBasedSpanComparator` since we also have a span comparator which is based on the start position.